engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/ValidatorEvaluator.scala

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.engine.definition.validator
+
+import java.lang.reflect.Parameter
+
+import pl.touk.nussknacker.engine.api.definition.ParameterValidator
+
+trait ValidatorEvaluator {
```

Comment:
Can you name it `ValidatorExtractor`? We have a lot of lazy paramerters, interpreters and you word "evaluator" when we need to evaluate value of expression.  Here you extract some information from method definition. It is very similiar to `TypesInformationExtractor`

Reply:
Done